File: test/Movie/new/test04.c
#include <libming.h>

static char *test = "test04";

main()
{
SWFMovie m;

m= newSWFMovieWithVersion(6);
SWFMovie_setBackground(m, 0xff, 0xff, 0xff);
SWFMovie_save(m,"test04.swf");

return 0;
}

File: test/Movie/new/test04.py
#!/usr/bin/python
from ming import *

Ming_useSWFVersion(6);

m =  SWFMovie();


#  SWF_SETBACKGROUNDCOLOR 
m.setBackground(0xff, 0xff, 0xff);
m.nextFrame();  # end of frame 1

m.save("test04.swf");
